spec/client/ics-007-tendermint-client/README.md

```diff
@@ -278,22 +281,31 @@ function checkForMisbehaviour(
           return true
         }
       case Misbehaviour:
-        // assert that the heights are the same
-        assert(misbehaviour.h1.height === misbehaviour.h2.height)
-        // assert that the commitments are different
-        assert(misbehaviour.h1.commitmentRoot !== misbehaviour.h2.commitmentRoot)
+        if (misbehaviour.h1.height < misbehaviour.h2.height) {
+          return false
+        }
```

Comment:
@AdityaSripal I added this check to [follow ibc-go](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/misbehaviour.go#L87) as we discussed. If this is ok, then I'll merge.